src/js/BookReader.js

```diff
@@ -581,6 +581,21 @@ BookReader.prototype.drawLeafs = function() {
   }
 };
 
+BookReader.prototype.leafTemplate = function(styles, index) {
```

Comment:
Also, let's make this private. Since this repo is also used by third parties, I don't want to have to support this function being public unless we need it to be :P

```suggestion
/**
 * @private
 */
BookReader.prototype._leafTemplate = function(styles, index) {
```